### PR TITLE
Fix #736 : User Profile UI Issue using Sweet Error Library[Offline Mode]

### LINKED ISSUE #736: Improve UI Design of User Profile Fragment
**Summary:**

Improve UI Design of User Profile Fragment in Offline mode

**Steps to reproduce:**

Click on User Image either from Nav Drawar or from Home Fragment

**Expected behaviour:**

Improper change of visibility, and activity should refresh on clicking the Error Icon (like done in other activities)

**Observed behaviour:**

Unwanted view are visible, clicking Error Icon does nothing

**Device and Android version:**

Xiaomi, Marshmallow

**Screenshots:**

issue_user_profile_offline_mode

### PR DESCRIPTION
**Pull request by miPlodder**
Fixes #736 

screenshot_2018-05-06-22-01-03-688_org mifos mobilebanking

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by miPlodder** [comment]
@therajanmaurya @dilpreet96 @droidchef @satyan Please review this PR.

**Comment by therajanmaurya** [comment]
@miPlodder Please use sweet error library UI.

**Comment by miPlodder** [comment]
@therajanmaurya Please review the PR again, I have made the requested changes.

**Comment by miPlodder** [comment]
@therajanmaurya Please review this PR again, I have made the requested changes.

File: app/src/main/java/org/mifos/mobilebanking/ui/fragments/UserProfileFragment.java [hunk 1]
```diff
@@ -209,9 +212,8 @@ public void run() {
     @Override
     public void showError(String message) {
         Toaster.show(rootView, message);
-        appBarLayout.setVisibility(View.GONE);
-        llUserProfile.setVisibility(View.GONE);
-        rlError.setVisibility(View.VISIBLE);
+        sweetUIErrorHandler.showSweetCustomErrorUI(getString(R.string.error_message_server),
```
Comment (therajanmaurya) [inline app/src/main/java/org/mifos/mobilebanking/ui/fragments/UserProfileFragment.java]: Error message should be `Failed to fetch user profile`

**Review by therajanmaurya — CHANGES_REQUESTED** [review]

**Comment by miPlodder** [comment]
@therajanmaurya I have made the requested changes. Please review again.

**Review by therajanmaurya — APPROVED** [review]

**Pull request merged**